binding/Binding/SKDrawable.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+
+namespace SkiaSharp
+{
+	public abstract class SKDrawable : SKAbstractDrawable
```

Comment:
Because there is a `SKDrawable` from the native code, we should rather name this `SKManagedDrawable` to both match the C++ types, but also so we can keep the inheritance the same.

We can add a `SKDrawable` that is also abstract (so that we cannot new it up) and an internal constructor (so that we can prevent users from creating any direct types) - they need to use the `SKManagedDrawable`.